homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/arrayservice/ArrayService.java

```diff
@@ -0,0 +1,53 @@
+package com.tutrit.java.quickstart.homework.arrayservice;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ArrayService {
+    Logger log = LoggerFactory.getLogger("ArrayService");
+
+    public int[] create(int size) {
+        return new int[size];
+    }
+
+    public void fillRandomly(int[] array) {
+        for (int i = 0; i < array.length; i++) {
+            array[i] = (int) (Math.random() * 100.1);
```

Comment:
I don't can it review. I need see the description your task.

Reply:
fixed